docker-compose.yml

```diff
@@ -34,18 +38,22 @@ volumes:
   zoo:
 
 networks:
-  internal:
-
   default:
     name: stackcar
 
 services:
+  adminer:
+    extends:
+      file: hyrax-webapp/docker-compose.yml
+      service: adminer
+
   zoo:
     extends:
       file: hyrax-webapp/docker-compose.yml
       service: zoo
 
   solr:
+    image: ghcr.io/notch8/palni_palci_knapsack/solr:${TAG:-latest}
```

Comment:
With the updated actions we need to have a solr image for tests to pull and run in ci. We typically do not deploy solr images unless in a docker production environment.